tests/pos/i10080.scala

```diff
@@ -0,0 +1,5 @@
+trait Foo:
+end Foo
+
+trait Bar
+end Bar
```

Comment:
This one looks confusing to me because I read `trait Bar` as a complete definition, so there is nothing to end. As I’ve mentioned in the linked issue, to me the “brace-significant” equivalent would be:

~~~ scala
trait Bar
}
~~~

So, I’m not sure we should support it.

Also, it suggests that we could insert definitions between `trait Bar` and `end Bar`, like so:

~~~ scala
trait Bar
  def quux: Quux
end Bar
~~~

But this would be rejected, right?

Reply:
Yes, that one would be rejected.